### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Gelma**
Thanks a lot for your work.

This is just a stupid patch to fix assorted typos all around.

It passed the usual "make -j && make checkstyle && make lint"

### CODE REVIEW

File: module/avl/avl.c [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ avl_destroy_nodes(avl_tree_t *tree, void **cookie)
 	--tree->avl_numnodes;
 
 	/*
-	 * If we just did a right child or there isn't one, go up to parent.
+	 * If we just made a right child or there isn't one, go up to parent.
```
Comment (ahrens) [inline module/avl/avl.c:1011]: I don't think we are "making" any nodes here; "did" seems ok to me, or we could change it to "removed"

File: module/zstd/lib/zstd.c [hunk 2]
```diff
@@ -21273,7 +21273,7 @@ ZSTD_initStats_ultra(ZSTD_matchState_t* ms,
     ms->window.lowLimit = ms->window.dictLimit;
     ms->nextToUpdate = ms->window.dictLimit;
 
-    /* re-inforce weight of collected statistics */
+    /* re-enforce weight of collected statistics */
```
Comment (ahrens) [inline module/zstd/lib/zstd.c:21276]: this one isn't clear to me which way it should be... are we "enforcing" some sort of rule? or are we "reinforcing" (supplementing, increasing) the weight?  Maybe better to chose a different word altogether.

File: tests/zfs-tests/tests/functional/replacement/attach_multiple.ksh [hunk 3]
```diff
@@ -44,7 +44,7 @@ function cleanup
 	rm -f ${VDEV_FILES[@]}
 }
 
-log_assert "Verify attach/detech with multiple vdevs"
+log_assert "Verify attach/detect with multiple vdevs"
```
Comment (ahrens) [inline tests/zfs-tests/tests/functional/replacement/attach_multiple.ksh:47]:
```suggestion
log_assert "Verify attach/detach with multiple vdevs"
```

File: tests/zfs-tests/tests/functional/replacement/attach_multiple.ksh [hunk 4]
```diff
@@ -108,4 +108,4 @@ for replace_mode in "healing" "sequential"; do
 	log_must zpool wait $TESTPOOL1
 done
 
-log_pass "Verify attach/detech with multiple vdevs"
+log_pass "Verify attach/detect with multiple vdevs"
```
Comment (ahrens) [inline tests/zfs-tests/tests/functional/replacement/attach_multiple.ksh:111]:
```suggestion
log_pass "Verify attach/detach with multiple vdevs"
```

**Review by ahrens — COMMENTED** [review]
Thanks for fixing all of these!

File: module/zstd/lib/zstd.c (hunk 2 above)
Comment (freqlabs) [reply module/zstd/lib/zstd.c:21276]: We should prefer making any changes to zstd.c and zstd.h through upstream.

**Comment by behlendorf** [comment]
Merged after making the suggested correction from the review comments and dropping the changes to the upstream zstd code.

**Comment by Gelma** [comment]
> Merged after making the suggested corrections from the review comments and dropping the changes to the upstream zstd code.

Really thanks a lot. Crazy days for me, so I was late!

**Pull request closed without merging**